test/collections/VectorTest.ooc

```diff
@@ -214,12 +220,27 @@ VectorTest: class extends Fixture {
 		*/
 
 		this add("nested heap vector", func {
-			sizeX := 2
-			sizeY := 3
-			heapVector := HeapVector<HeapVector<MyCover>> new(sizeX)
-			for (i in 0 .. sizeX)
-				heapVector[i] = HeapVector<MyCover> new(sizeY)
-			heapVector free()
```

Comment:
The code you removed uses a `HeapVector<HeapVector<MyCover>>`, the new code uses a `HeapVector<HeapVector<MyClass>>`. Don't you think we should test both?